docs/source/en/perf_train_cpu_many.md

```diff
@@ -303,9 +304,9 @@ transformers-pytorchjob-worker-3                         1/1     Running
 ...
 ```
 
-The logs for worker can be viewed using `kubectl logs -n kubeflow <pod name>`. Add `-f` to stream the logs, for example:
+The logs for worker can be viewed using `kubectl logs <pod name>`. Add `-f` to stream the logs, for example:
```

Comment:
Maybe add `-n <namespace>`? or a note that says if you can't find your logs be sure to add `namespace`? Just a thought

Reply:
I actually added `-n <namespace>` to all the commands at first, but then removed it thinking that someone running this example probably knows how to use namespaces, and putting `-n <namespace>` can make the commands less copy/paste-able.

Reply:
Maybe I'll make it `-n=${NAMESPACE}` so that if anything it can be defined as a var and still be copy/pasted